src/data/markdown/translated-guides/en/02 Using k6/12 Execution context variables.md

```diff
@@ -47,6 +47,9 @@ Current VU number in use. The value is assigned incrementally for each new VU in
 >
 >In k6 Cloud you will also have additional [environment variables](/cloud/creating-and-running-a-test/cloud-tests-from-the-cli/#environment-variables) that will tell you on which server, load zone and distribution of the test you are currently executing.
 
+> ### # # ✨ Execution API
```

Comment:
This should prominent and at the top of the document, not somewhere in the middle. Ideally, we should rewrite the whole page, since we don't want anyone to use `__VU` and `__ITER` ever again... But that can wait for after the release, we can just add an issue about it, so we don't forget it.